packages/ag-charts-community/src/chart/series/cartesian/cartesianSeries.ts

```diff
@@ -47,6 +47,9 @@ export interface ErrorBoundSeriesNodeDatum {
     // by the size of the marker (line, scatter), width of the bar (vertical
     // bars), or height of the bar (horizontal bars).
     readonly capDefaults: { lengthRatio: number; lengthRatioMultiplier: number; lengthMax: number };
+    // The error bar module needs to know which node data to retrieve when
+    // events on are triggered on a datum.
+    readonly index: number;
```

Comment:
Can we discuss the use of `index` for this use case - we haven't needed it for any other cases, so I'm a little suspicious that this is needed?

Reply:
The `index` is required for the `ErrorBar` class to know which `ErrorBarNode` scene object to update.

However, I've removed this property. Instead, the `ErrorBar.restyleHightlightChange` method will determine the index by searching sequentially for a matching `datum` object.